powerauth-fido2-tests/lombok.config

```diff
@@ -0,0 +1 @@
+lombok.log.fieldName=logger
```

Comment:
🤔 This is the second lombok config in this repo. May a single one in the root could be enough.